src/exoplanet_core/pymc4/__init__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+__all__ = ["ops"]
+
+
+def __set_compiler_flags():
+    import aesara
+
+    def add_flag(current, new):
+        if new in current:
+            return current
+        return f"{current} {new}"
+
+    current = aesara.config.gcc__cxxflags
+    current = add_flag(current, "-Wno-c++11-narrowing")
+    current = add_flag(current, "-fno-exceptions")
+    current = add_flag(current, "-fno-unwind-tables")
+    current = add_flag(current, "-fno-asynchronous-unwind-tables")
```

Comment:
Were those required?

Reply:
Yeah - they are because PyMC (not Aesara) is where they are set otherwise. Therefore this code wouldn't work without either setting these ourselves or importing PyMC. But PyMC is not a required dependency and I wanted it to run regardless!